client/blocks/login/index.jsx

```diff
@@ -45,17 +46,27 @@ export class Login extends Component {
 	}
 
 	onChangeField( event ) {
-		this.setState( {
-			[ event.target.name ]: event.target.value
-		} );
+		const target = event.target;
+		const value = target.type === 'checkbox' ? target.checked : target.value;
+		const name = target.name;
+
+		if ( name === 'rememberme' ) {
+			this.props.recordTracksEvent( 'calypso_loginblock_rememberme_change', { new_value: value } );
+		}
+
+		this.setState( { [ name ]: value } );
 	}
```

Comment:
Another approach to avoid all these conditions could be to introduce an event handler only for the checkbox:

```javascript
onChangeField( event ) {
  const { name, value } = event.target;

  this.setState( { [ name ]: value } );
}

onRememberMeChange = ( event ) => {
  const { name, checked } = event.target;

  this.props.recordTracksEvent( 'calypso_loginblock_rememberme_change', { new_value: checked } );

  this.setState( { [ name ]: checked } );
};
```

```javascript
<FormCheckbox
  name="rememberme"
  checked={ this.state.rememberme }
  onChange={ this.onRememberMeChange }
  { ...isDisabled } />
```